server/src/modules/answers/answers.controller.ts

```diff
@@ -29,16 +30,10 @@ export class AnswersController {
    * @returns 200 with array of answers
    * @returns 500 if database error occurs
    */
-  listAnswers: ControllerHandler<
-    { id: string },
-    | {
-        body: string
-        username: string
-        userId: number
-        agencyLogo: string
-      }[]
-    | Message
-  > = async (req, res) => {
+  listAnswers: ControllerHandler<{ id: string }, Answer[] | Message> = async (
```

Comment:
If we are going for agency-centric question and answer, is there still a need for value returned to be an array?

Reply:
I'm not entirely sure. We _might_ want to return multiple answers for other reasons (perhaps if we want to return answers and updates to such answers over time?). The changes needed to frontend are a little invasive anyway, so perhaps we can defer the changes to response shape to a later PR.

Reply:
Hm okay maybe this in its current form for this PR is fine. There doesn't seem to be a way for agency to input more than one answer, so it seems a little inconsistent in the codebase. Maybe in the future this will be useful then :+1

Reply:
fwiw, I think the answers and updates over time might be confusing for users, but this is separate discussion haha